src/uncrustify_emscripten.cpp

```diff
@@ -363,7 +353,7 @@ string show_config()
 void _initialize()
 {
    register_options();
-   set_option_defaults();
+   //set_option_defaults();
```

Comment:
BTW, is this still needed? ISTM this could just be deleted, rather than being commented out.